Ricochet/Assets/_Scripts/Player/PlayerController.cs

```diff
@@ -280,6 +287,16 @@ public void PlayerDead()
         rigid.velocity = Vector3.zero;
         gameObject.SetActive(false);
     }
+
+    public void EnableSecondaryShield(bool status)
+    {
+        switch (currPowerUp)
```

Comment:
This already is checking in the game manager before its called.  Don't need to switch on the current power up

Reply:
I had this in-case we would have multiple secondary shields and it would then use this case switch to figure out which one should be activated